lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -473,17 +475,11 @@ def test_grade_with_max_score_cache(self):
                 module_state_key=self.problem_location('p2')
             ).exists()
         )
-        location_to_cache = unicode(self.problem_location('p2'))
-        max_scores_cache = grades.MaxScoresCache.create_for_course(self.course)
 
-        # problem isn't in the cache
-        max_scores_cache.fetch_from_remote([location_to_cache])
-        self.assertIsNone(max_scores_cache.get(location_to_cache))
```

Comment:
I simplified this test a bit here - I'm not sure how to peek into the course_blocks cache to assert that `max_score` is cached or not, and I'm not entirely convinced that this test i the correct location for those checks.

I've updated the docstring to reflect this new, weaker assertion being tested.